src/main/webapp/jsp/studentCourseDetails.jsp

```diff
@@ -9,7 +9,7 @@
   <script type="text/javascript" src="/js/studentCourseDetails.js"></script>
 </c:set>
 
-<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="TEAMMATES - Student" jsIncludes="${jsIncludes}">
+<ts:studentPage bodyTitle="Team Details for ${data.studentCourseDetailsPanel.courseId}" pageTitle="Team Details" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the title would include the course ID.